targets/unity-v2/source/Shared/Public/PluginContractKey.cs

```diff
@@ -1,11 +1,8 @@
-using System;
 using System.Collections.Generic;
-using PlayFab.Internal;
-using PlayFab.Json;
 
 namespace PlayFab
 {
-    public class PluginContractKey
+    public struct PluginContractKey
```

Comment:
Software is an art. I love fixes like this!

Reply:
I only care whether the Unity GC allocator deep inspector reports GC allocations or not, and if I change it to a struct, it says zero bytes.
It's still invoking a constructor and allocating memory, but it's doing so in stack memory instead of heap memory, which means it's not hitting the garbage collector.
To my view, that's "Good enough".
I feel that on the "clean maintainable code" vs "optimal performance", newing a struct is the best mix. Everybody understands what the code does, and everybody understands what decisions are being made.